set.mm

```diff
@@ -490300,9 +490358,9 @@ strict in the case where the sets B(x) overlap.  (Contributed by Thierry
         AVVNXKVUFVUKVVRVWNYSXMVUIVXGFVVJUYDVUGVVJVPVUHVWPRVUGVVJUXTVQVIXNXOAJVW
         PVWCKLMNUULXKXQUUMUUNYFXQVUPUYCRUUSSZUIZUWJRUYCXHUUOULZUKULZUIZVUQVURUU
         PZVUDVXLVUEVUKUYMVUOVUDUYCUUTVXKUEUXOUUQUURUVAUVBVUPVUFVUOVXOVVEVUNVUOU
-        TVUFVUOVXOVUFUYRVXNUWJVUFUYQVXMRUKUEUXOUYGUVCVKWFXJXOVXLVXOVXPRUYCUWJUV
-        DXJXOUVHUVEUVFUVGUVIUVJUVKUWLUWIUAHUWAUWJHVPUWKUWHRUWJHUWGVQVIXNXOUVL
-        $.
+        TVUFVUOVXOVUFUYRVXNUWJVUFUYQVXMRUKVUDUYQVXMVPVUEUEUXOUYGUVCYFVKWFXJXOVX
+        LVXOVXPRUYCUWJUVDXJXOUVHUVEUVFUVGUVIUVJUVKUWLUWIUAHUWAUWJHVPUWKUWHRUWJH
+        UWGVQVIXNXOUVL $.
```

Comment:
> You seem to actually have made this proof longer - am I correct?

Yes, you are right, because I made some ccat* theorems more general. Maybe the proof can become shorter if it is revised manually (I only changed the modified theorems, and ran minimize afterwards).

Reply:
Oh, I see: you changed the ccat* theorems in main, and that proof in my mathbox uses them, so it had to be adapted. Ok then, this is fine to me!